src/feed/tasks.py

```diff
@@ -166,51 +175,132 @@ def refresh_feed_hot_scores():
         return False
 
     try:
-        _refresh_feed_hot_scores()
+        # Uses default 30-day lookback for papers and posts only
+        stats = refresh_feed_hot_scores_batch(
+            queryset=None,  # Will auto-filter to last 30 days
+            update_v1=True,
+            update_v2=True,
+            content_types=None,  # Defaults to paper and post
+        )
+        logger.info(f"Refreshed hot scores: {stats}")
+        sentry.log_info(f"Refreshed hot scores: {stats}")
+        return stats
     finally:
         lock.release(key)
         logger.info(f"Released lock {key}")
 
 
-def _refresh_feed_hot_scores():
+def refresh_feed_hot_scores_batch(
```

Comment:
A bit more verbose but faster, more robust, and can be used in multiple spots (e.g. backfill script)